File: java/spring-boot-dynamodb/src/main/java/com/meridal/examples/springbootdynamo/repository/RecordingRepository.java
package com.meridal.examples.springbootdynamo.repository;

import com.meridal.examples.springbootdynamo.domain.Recording;
import org.socialsignin.spring.data.dynamodb.repository.EnableScan;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
@EnableScan
public interface RecordingRepository extends CrudRepository<Recording, String> {
}

File: kotlin/kotlin-spring-boot-dynamodb/src/main/kotlin/com/meridal/examples/kotlinspringboot/repository/RecordingRepository.kt
package com.meridal.examples.kotlinspringboot.repository

import com.meridal.examples.kotlinspringboot.model.Recording
import org.socialsignin.spring.data.dynamodb.repository.EnableScan
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
@EnableScan
interface RecordingRepository: CrudRepository<Recording, String> {
}